versions.yaml

```diff
@@ -84,7 +84,7 @@ assets:
       uscan-url: >-
         https://github.com/intel/nemu/tags
         .*/release-?(\d\S+)\.tar\.gz
-      version: "release-2019-03-11"
+      version: "pre-release-2019-04-100"
```

Comment:
any particular reason this is called pre-release? If that is just so we can fire this PR through the CI system, fine :-) But, for a merged release, that doesn't feel great.

Reply:
This is only for a CI test. I don't expect this to be merged until we also make a stable release. Unfortunately it is a limitation of the NEMU Kata CI that it can't take from source otherwise we wouldn't have needed to make a "release" at all.

Reply:
Thanks @rbradford. Sure, Kata CI was never (so far) designed to take NEMU from source.
Best put a do-not-merge label/title on test PRs (thanks @sboeuf for adding one here already), or they could accidentally get merged :-)